lib/ansible/modules/files/lineinfile.py

```diff
@@ -202,6 +202,67 @@
     validate: /usr/sbin/visudo -cf %s
 '''
 
+RETURN = r'''
+changed:
+  description: True if the file was changed during a task execution.
+  returned: always
+  type: bool
+  sample: False
+msg:
+  description: Descriptive status of a task result.
+  returned: When changed a status message otherwise an empty string.
+  type: str
+  sample: line added
+backup:
+  description: The full name of the backup file created.
+  returned: When backup parameter is set to True, otherwise it returns an empty string.
+  type: str
+  sample: '/etc/services.2243.2020-02-07@20:46:55~'
+diff:
+  description: The differences made by the task execution.
+  returned: on success
+  type: complex
+  contains:
+    - after:
+        description: The content of the changed file after a change. The content is converted to string.
+        returned: When the parameter `--diff` is used otherwise empty string is returned.
```

Comment:
Module doc strings are restructured text format, so double backticks are needed for inline code snippets.
```suggestion
        returned: When the parameter ``--diff`` is used otherwise empty string is returned.
```